### DISCUSSION TITLE
How does one code generator obtain member information generated by another code generator?

### DISCUSSION THREAD

**Discussion by weiyoucheng** [dotnet/roslyn#74711] [Q&A]
![image](https://github.com/user-attachments/assets/8cccdb11-143f-42b7-8bef-c0d6cc81bced)
I would like to obtain properties generated by the CommunityToolkit.Mvvm toolkit to implement serialization methods for the MemoryPack interface. However, I cannot see the member information generated by the CommunityToolkit.Mvvm toolkit in the semantic member list of that type. Are there any ways for me to access the member information generated by the CommunityToolkit.Mvvm toolkit?

**Comment by 333fred** [comment]
No, there are no ways to do this. Generators all run at the same time, they cannot see each other's output.

**Reply by weiyoucheng** [reply]
If there were a way to set the execution order, that would be great. For example, specifying that the "B" code generator should run after the "A" generator, and if the order is not specified, it would default to parallel processing.

**Reply by 333fred** [reply]
See the discussion in https://github.com/dotnet/roslyn/issues/57239.
